armory/datasets/package.py

```diff
@@ -18,16 +18,30 @@ def package(
     """
     Package a built dataset as .tar.gz, return path
     """
-    version, data_dir, built_data_dir, subdir = build.build_info(
+    version, data_dir, built_data_dir, subdir, builder_configs = build.build_info(
         name, version=version, data_dir=data_dir
     )
 
     data_dir = Path(data_dir)
-    expected_dir = data_dir / name / version
-    if not expected_dir.is_dir():
-        raise FileNotFoundError(f"Dataset {name} not found at {expected_dir}")
-    tar_full_filepath = common.get_cache_dataset_path(name, version)
 
+    if not builder_configs:
+        tar_list = [str(Path(name) / version)]
+    else:
+        # including .config, which contains metadata.json that tfds refers to for default_config_name of a given dataset
+        # .config and metadata.json are hardcoded values for tfds v4.6.0
```

Comment:
instead of hardcoding `".config"` in a few places can we create a constant variable at top of file?

Reply:
@ppark-twosixtech understood that these are hardcoded on the tfds side, but setting them as constant variables in Armory may make this a little easier to maintain

Reply:
resolved with f146e0b2